use-cases/using-SRIOV-and-kata.md

```diff
@@ -41,6 +41,10 @@ In order to setup your host for SR-IOV, the following has to be true:
   Controller (NIC). The section [Checking your NIC for SR-IOV](#checking-your-nic-for-sr-iov) describes how to assess if you need to make NIC changes and how to make
   the necessary changes.
 
+Besides, you need to enable the NIC driver in your guest kernel config:
+eg. mlx5 for Mellanox NIC.
```

Comment:
I'd rewrite it as following:

Besides, you need to enable the NIC driver in your guest kernel config (e.g. `mlx5` for Mellanox NIC).

Reply:
thanks, updated